.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "trailingComma": "none",
   "singleQuote": true,
-  "printWidth": 100
+  "printWidth": 100,
+  "endOfLine": "auto"
```

Comment:
needed for windows

Reply:
what's the issue? looks like this option infers the line ending from the first one found in the file, so we could have mixed lf/crlf files in the codebase.

Reply:
All the npm run lint commands will fail on windows without this change

Reply:
Gotcha, confirmed this myself just now on my Windows PC. Nice catch!